contracts/Staking.sol

```diff
@@ -2,46 +2,56 @@ pragma solidity 0.6.2;
 
 import "./StorageManager.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @title staking
 /// @author see ERC900 proposal
 /// @author Rinke Hendriksen <rinke@iovlabs.org>
 /// @notice implements the ERC900 interface, with some small modifications. The contract also handles native tokens.
-contract Staking {
+contract Staking is Ownable {
 
     using SafeMath for uint256;
 
     StorageManager storageManager;
```

Comment:
```suggestion
    StorageManager public storageManager;
```

Reply:
I would make the `storageManager` public and lets create `function setStorageManager(address) public onlyOwner` so if for some reason we migrate to another Storage contract we can redirect the Staking to that contract.

Reply:
Done

Reply:
Nice point 👍